ddp-workspace/projects/ddp-osteo/src/styles/activity.scss

```diff
@@ -319,6 +324,12 @@ select:disabled {
     }
 }
 
+.ddp-content__agreement {
+    font-family: 'Source Serif Pro', serif;
+    font-size: 1.5rem;
+    line-height: 2.875rem;
+}
+
```

Comment:
Need some backend fixes to support another style https://github.com/broadinstitute/ddp-study-server/pull/11
![image](https://user-images.githubusercontent.com/30126907/71069250-61025380-2189-11ea-8212-8d6110dd3d75.png)